src/core/talent.js

```diff
@@ -118,12 +120,16 @@ class Talent extends IOFeatures {
     }
 
     start() {
+        if (this.__getRules() === null) {
+            throw new Error('The talent must provide some event rules!')
+        }
+        const talentTopic = Talent.getTalentTopic(this.id, jsonHash(this.__getRules().save(true)))
```

Comment:
What happens if the getRules() are changed at runtime? --> in this case the talent has to 'connect' to a different topic, right?